dclab/kde/base.py

```diff
@@ -127,8 +127,8 @@ def get_contour(self, xax="area_um", yax="deform", xacc=None, yacc=None,
 
     def get_contour_lines(self, xax="area_um", yax="deform", xacc=None,
                           yacc=None, kde_type="histogram", kde_kwargs=None,
-                          xscale="linear", yscale="linear", ret_levels=False,
-                          quantiles=None):
+                          xscale="linear", yscale="linear", quantiles=None,
```

Comment:
Please put quantiles as the **very first** argument. This is the important parameter that everyone will look for first.